bootloader/wscript

```diff
@@ -411,7 +411,7 @@ def configure(ctx):
         ctx.msg('System', 'Assuming cross-compilation for %s' %
                 DESTOS_TO_SYSTEM[ctx.env.DEST_OS])
 
-        if ctx.env.DEST_OS in ('freebsd', 'hpux', 'sunos'):
+        if ctx.env.DEST_OS in ('hpux', 'sunos'):
             # For these FreeBSD and HP-UX we determine some settings from
```

Comment:
Can you also amend the corresponding comment? ("For these systems" or something along those lines...)